contracts/airdrops/Existing1155NftDrop.sol

```diff
@@ -139,40 +101,24 @@ contract Existing1155NftDrop is AirdropInfo, AirdropMerkleProof, IERC1155Receive
         rewardToken.safeTransferFrom(address(this), msg.sender, rewardTokenId, tokensPerClaim * tokenIds.length, "");
     }
 
-    //@notice Get the type of airdrop, it's either ERC20, ERC721, ERC1155
-    function getAirdropType() external pure override returns (string memory) {
-        return "ERC1155";
-    }
-
-    //@notice Checks if the user is eligible for this airdrop
-    function isEligibleForReward(uint256 tokenId) external view returns (bool) {
+    /// @notice Checks if the user is eligible for this airdrop
+    function isEligibleForReward(uint256 tokenId) public view returns (bool) {
         if (hasClaimed[tokenId]) revert AlreadyRedeemed();
+        if (rewardedNft.ownerOf(tokenId) != msg.sender) revert Unauthorized();
```

Comment:
I don't understand this.. 
User is supposed to find out if he is eligible for reward, 
and here we are checking if he owns `rewardedNft` and revert if he doesn't

Reply:
In order to be eligible for reward, the user must own the rewardedNfr

Reply:
My bad, brain afk on friday :))

Reply:
No worries, 17:00 is nearly here! 🎉